Final_project_2/frontend/css/style.css

```diff
@@ -463,20 +463,15 @@ label  {
   margin-right: 30px;
 } 
 
-
-
 #contactform {
-  padding: 80px;
-  
+    display: flex;
+    padding-left: 80px;
+    padding-right: 80px;
 }
 
-.fanmail {
```

Comment:
the styles in the fanmail block weren't used or weren't necessary.

You can see this when you inspect an element in your bworser: note how the `flex-direction` property is muted in color and there's the little `i` bubble next to it (hovering on it will tell you more information about why this CSS property is unused):

<img width="197" alt="image" src="https://github.com/deimantei/Women_Go_Tech/assets/32952943/d15bce8c-6011-4396-897d-fa4af7eb9e77">